lib/database/models/lhcPeriodsStatistics.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const Sequelize = require('sequelize');
+
+module.exports = (sequelize) => {
+    const LhcPeriodStatistics = sequelize.define(
+        'LhcPeriodForPhysicsRuns',
+        {
```

Comment:
Try to be consistent in your naming!
`LhcPeriodStatistics`, `LhcPeriodForPhysicsRuns` and `lhc_periods_with_statistics`... (I personally prefer the first one)

Reply:
done